ansible/roles/etcd/tasks/main.yml

```diff
@@ -11,4 +11,4 @@
 
 - name: Call restart etcd2
   include: etcd2_restart.yml
-  when: etcd2_modified == true and etcd_started.changed == false
+  when: etcd2_modified == true and etcd2_started.changed == false
```

Comment:
@ingvagabund I see where you want to take this for service wide variables, but I think keeping etcd2 might be more readable, since all the file names for the CoreOS etcd story is under etcd2. If you are looking for service wide variable names, the etcd role should be refactored to have one etcd-start.yml, one etcd-stop.yml.... and within those have all the statements contain the conditional `is_coreos` to distinguish between the etcd plays. I thought having a bunch of statements with identical conditionals was verbose.

I broke them down in my original refactor because I thought it was better to simplify all playbooks and make it easy to separate the different processes. I fought with that decision for a while and could see it going either way, single `etcd-started.yml` files or separate.


Visually this LGTM. I did not pick this error up in my virtualenv, I believe it's because I'm doing a clean vagrant up every time.